style.css

```diff
@@ -0,0 +1,130 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  position: relative;
+}
+
+.container {
+  width: 100vw;
+  min-height: 100vh;
+  position: relative;
+}
+
+nav {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  padding: 20px 30px;
+}
```

Comment:
- [ ] **[OPTIONAL]:** I noticed that you have multiple repeated CSS properties across your stylesheet. It would be great if you can find a way of reducing this duplication of code ♻️. Remember that not repeating pieces of code will keep your codebase clean and readable. 🤓 🛠️ 

From [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md):
![image](https://user-images.githubusercontent.com/77069011/200000947-9e58c4d8-fed4-45f8-a822-7828c6e713f7.png)


In this case, you could do this with flex:

```CSS
.flex {
  display: flex;
}
``` 

And then, use it in your HTML elements where you need a flex container:

```HTML
<div class="flex"> This is a flex container </div>
```

Please, consider fixing this! 🤓